pkg/minikube/node/start.go

```diff
@@ -196,7 +196,7 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 		}
 	}
 
-	klog.Infof("Will wait %s for node ...", waitTimeout)
+	klog.Infof("Will wait %s for node up to ", viper.GetDuration(waitTimeout))
```

Comment:
Not sure what "up to" is trying to convey here

Reply:
before this it was getting the "flag-name"
this means will wait up to this amount of time for the cluster to be up